config/database.yml

```diff
@@ -5,6 +5,8 @@ development:
   url: <%= ENV.fetch('pg_url', 'postgres://localhost') %>
   database: dcaf_case_management_development
   pool: 5
+  username: <%= ENV['POSTGRES_USER'] || `whoami` %>
```

Comment:
I'd default this to `postgres` unless there's a specific reason not to.

Reply:
Could you say why?